test/e2e/atlas/streams_test.go

```diff
@@ -0,0 +1,279 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && debug)
```

Comment:
also realizing this is probably not running in our CI, I don't think we account for this tag `debug`

You probably want your own build variant for streams here https://github.com/mongodb/mongodb-atlas-cli/blob/2d2c2e8133997e25715d3e182e00f884abbd3eef/build/ci/evergreen.yml#L1413 and any related task

Reply:
Ah, forgot to remove this, I had this purely because I was having issues getting VSCode to run these tests (I missed the explanation in the README.md until the very end), so I used this to skip the other tests while I was iterating. I'll fix this.

Reply:
Addressed in [f5ec472](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/f5ec47231f56365f0171e5d254b7094f950a41d6)